BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1488,7 +1473,7 @@ IEnumerator FireReloadableMissile()
             }
 
             ml.launched = true;
-            var wpm = VesselModuleRegistry.GetMissileFire(SourceVessel, true);
+            var wpm = SourceVessel.ActiveController().WM;
             ml.SourceVessel = SourceVessel;
```

Comment:
Same for ReloadableFire, though this also gets into the same question as in ModuleWeapon of do we want a mother craft selecting/launching child craft missiles while child vessels are still docked?

Reply:
Hmm, that has some `yield`s while it's spawning the missile. We could grab the WM at the start of `FireReloadableMissile` and abort if it's changed at this point, perhaps?

Reply:
That could work.